README.md

```diff
@@ -153,6 +153,21 @@ abstract class NativeAutomator<IOSServer, AndroidServer, DartClient> {
 
 3. Run `gen_from_schema` script, few files will be updated
 
+## Develop patrol_cli
+
+If you have previously activated patrol_cli run:
+
+```bash
+dart pub global deactivate patrol_cli
+```
+
+then
+
+```bash
+cd patrol/packages/patrol_cli
```

Comment:
I'd say `cd packages/patrol_cli`, because it's best to assume that the working directory would be the README's one